### PR TITLE
Upgrade to Helix v0.19, React 16, and Storybook

### PR DESCRIPTION
**Pull request by nicko-winner**
Upgrade Helix to v0.19, React 16, and Storybook 5

**Storybook 3 (Before):**

**Storybook 5 (After):**

**todo:** 
- [x] Add some padding to default storybook container to match old storybook spacing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@storybook/react": "3",
     "babel-core": "6",
     "babel-runtime": "6",
-    "helix-ui": "^0.12.0",
+    "helix-ui": "^0.19.0",
     "prop-types": "15",
     "react": "15",
     "react-dom": "15"
```
Comment (100stacks) [inline package.json]: Should we update the other dev dependencies while we're at it?

```js
"devDependencies": {
    "@storybook/addon-actions": "3",	
    "@storybook/addon-knobs": "3",	
    "@storybook/addon-options": "3",
    "@storybook/addon-viewport": "3",
    "@storybook/addons": "3",
    "@storybook/react": "3",
    "babel-core": "6",
    "babel-runtime": "6",
    "helix-ui": "^0.12.0",
    "prop-types": "15",
    "react": "15",
    "react-dom": "15"
}
```

Comment (nicko-winner) [reply package.json]: sure we can do that.

**Comment by HelixUI** [comment]
Deploy preview for *helix-react* processing.

Building with commit c98edb50bc8d89e52acca4074ce54c31058cf407

https://app.netlify.com/sites/helix-react/deploys/5e77dc764fee5df20a560e90

**Comment by HelixUI** [comment]
Deploy preview for *helix-react* failed.

Built with commit c98edb50bc8d89e52acca4074ce54c31058cf407

https://app.netlify.com/sites/helix-react/deploys/5e77dc764fee5df20a560e90

File: .storybook/helixTheme.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+    base: 'light',
+    brandTitle: 'Helix React',
+    brandUrl: 'https://github.com/HelixDesignSystem/helix-ui',
```
Comment (100stacks) [inline .storybook/helixTheme.js]: Please change repo:

```js
brandUrl: 'https://github.com/HelixDesignSystem/helix-react',
```

Comment (nicko-winner) [reply .storybook/helixTheme.js]: good catch
```suggestion
    brandUrl: 'https://github.com/HelixDesignSystem/helix-react',
```

**Pull request merged**